plugin/trino-pinot/src/main/java/io/trino/plugin/pinot/query/DynamicTable.java

```diff
@@ -72,6 +74,8 @@ public DynamicTable(
         this.limit = requireNonNull(limit, "limit is null");
         this.offset = requireNonNull(offset, "offset is null");
         this.query = requireNonNull(query, "query is null");
+        this.isAggregateInSelectList = selections.stream()
```

Comment:
nit: `projections` instead of `selections`?

Reply:
sgtm!